shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-transaction-spring/src/test/java/org/apache/shardingsphere/spring/transaction/ShardingTransactionTypeScannerTest.java

```diff
@@ -42,4 +44,15 @@ public void assertShardingTransaction() {
         assertThat(mockService.executeBase(), is(TransactionType.BASE));
         assertThat(mockService.execute(), is(TransactionType.XA));
     }
+    
+    @Test
+    public void assertShardingTransactionType() {
+        TransactionType preTransactionType = TransactionTypeHolder.get();
+        mockService.executeLocal();
+        assertEquals(preTransactionType, TransactionTypeHolder.get());
```

Comment:
I think we can assertThat a precise value here, i.g. `asserThat(TransactionTypeHolder.get(), is(TransactionType.LOCAL))`.